inference/core/entities/requests/sam2.py

```diff
@@ -56,6 +56,61 @@ class Sam2EmbeddingRequest(Sam2InferenceRequest):
     )
 
 
+class Box(BaseModel):
+    x: float = Field()
+    y: float = Field()
+    width: float = Field()
+    height: float = Field()
+
+
+class Point(BaseModel):
+    x: float = Field()
+    y: float = Field()
+    positive: bool = Field()
+
+
+class Sam2Prompt(BaseModel):
+    box: Optional[Box] = Field(default=None)
+    points: Optional[List[Point]] = Field(default=None)
+
+
+class Sam2PromptSet(BaseModel):
+    prompts: Optional[List[Sam2Prompt]] = Field(default=None)
```

Comment:
Q: What does empty prompts set semantically mean?

Reply:
lets avoid mutations for payload entities if possible

Reply:
>Q: What does empty prompts set semantically mean?

If you don't prompt the model, it'll just return the mask of the object most in the foreground

Reply:
👍